core/templates/Polyfills.ts

```diff
@@ -121,9 +121,9 @@ if (!('outerHTML' in SVGElement.prototype)) {
 // so we set an empty object first.
 if (navigator.mediaDevices === undefined) {
   // This throws "Cannot assign to 'mediaDevices' because it
-  // is a read-only property." We need to suppress this since some browsers
-  // may not have this property at all. So, we need to set it to an empty
-  // object.
+  // is a read-only property.". We need to suppress this error because since
```

Comment:
Why you changed this?

Reply:
I changed this because it does not follow pattern required for '@ts-ignore' or '@ts-expect-error' which I am implementing in comment-style check
https://github.com/oppia/oppia/pull/13022/files#diff-ebdbd754a4a10a16d3a71e77edcaa4e50310e3e7e14fbb47a456f5d0d0b4d272R92-R105

Reply:
Ah -- I see. Thanks for the explanation. Just one suggestion.
```suggestion
  // is a read-only property.". We need to suppress this error because
```

Reply:
Done!